packages/forms/src/UIForm/fields/NestedListView/NestedListView.utils.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Build the ListView component props to match current state's values
```

Comment:
`component props to match current state's values` => `component props accordingly to state's values` ?

Reply:
Looks already clear to me